ScibConfig.py

```diff
@@ -75,16 +63,33 @@ def get_from_method(self, method, key):
         if key not in self.METHODS[method]:
             return False
             # raise ValueError(f"{key} not a valid attribute of scenario {scenario}")
-        return self.METHODS[method][key]
-
-    def get_hvg(self, wildcards, adata_path):
+        value = self.METHODS[method][key]
+        if key == 'output_type':
+            return value if isinstance(value, list) else [value]
+        return value
+
+    def get_hvg(
+            self,
+            wildcards: snakemake.io.Wildcards,
+            output_pattern: str = None,
+            **kwargs
+    ) -> str:
+        """
+        Get hvg parameter for integration run scripts
+        :param wildcards: wildcards passed by Snakemake containing at least 'hvg' key
+        :param output_pattern: file pattern with placeholders 'hvg'
+            Only needed for R integration methods to get path of separate HVG file
+            minimal example: 'output_dir/{hvg}.h5ad'
```

Comment:
I think we only pass a separate HVG file to Seurat, but not other R methods. Have you changed this to do so for all R methods?

Reply:
In the `integration_run_r` rule (check `scripts/integration/Snakefile`) I always pass a HVG file to the script. I'm assuming that script uses the HVGs for all R methods in question

Reply:
From what I can see, the count object is subset by HVGs for all R methods, if it is given as an option

Reply:
Hah... you're right... the subsetting is done later for R methods... only Seurat passes the list of HVGs. This is fine... was worried for a sec ^^.